src/blocks/card.scss

```
@@ -0,0 +1,90 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border-radius: 5px;
+  background-color: $color-white;
+  border: 1px solid $color-almostWhite;
+  padding: 32px 16px 16px ;
+
+  &__image {
+    height: 134px;
+    width: 160px;
+    margin: 0 3px 40px;
+  }
+
+  &__description {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin: 0 0 4px;
+    color: $color-main;
+  }
+
+  &__itemCode {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    height: 14px;
+    color: $color-secondary;
+    margin: 0 0 16px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    font-weight: 400;
+    font-size: 10px;
+    color: $color-main;
+    margin: 0;
+  }
+
+  &__price {
+    display: flex;
+    margin-bottom: 16px;
+    justify-content: space-between;
+  }
+
+  &__priceText {
+    color: $color-secondary;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 18px;
+    margin: 0;
+  }
+
+  &__priceValue {
+    color: $color-main;
+    text-align: right;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    margin: 0;
+  }
```

Need to use naming for classes with BEM and rewrite like this
```suggestion
  &__price {
    display: flex;
    margin-bottom: 16px;
    justify-content: space-between;
    
    &--text {}
    
    &--value {}
  }
```

---

done